core/src/reporter/api.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+import { Logger } from 'pino'
+import { OraklError, OraklErrorCode } from '../errors'
+import { ORAKL_NETWORK_API_URL } from '../settings'
+import { buildUrl } from '../utils'
+import { ISubmissionData } from './types'
+
+const LAST_SUBMISSION_ENDPOINT = buildUrl(ORAKL_NETWORK_API_URL, `last-submission`)
+
+export async function storeSubmission({
+  submissionData,
+  logger
+}: {
+  submissionData: ISubmissionData
+  logger?: Logger
+}) {
+  try {
+    const response = await axios.put(LAST_SUBMISSION_ENDPOINT, submissionData)
+    logger?.info('Reporter submission upserted', response.data)
+    return response.data
+  } catch (e) {
+    logger?.error(e.msg)
+    throw new OraklError(OraklErrorCode.FailedToUpsertSubmission)
+  }
+}
+
+export async function loadAggregatorByAddress({
```

Comment:
This is quite expensive operation to do during every submission. We will have to remove it soon. Please make an issue for it.